tests/unit/src/test/scala/tests/decorations/SyntheticDecorationsLspSuite.scala

```diff
@@ -487,13 +488,13 @@ class SyntheticDecorationsLspSuite extends BaseLspSuite("implicits") {
       _ <- server.didSave("a/src/main/scala/Main.scala")(identity)
       _ = assertNoDiagnostics()
       _ = assertNoDiff(
-        client.workspaceDecorations,
+        client.syntheticDecorations,
         """|object Main{
            |  val head: Double :: tail: List[Double] = List[Double](0.1, 0.2, 0.3)
            |  val List[Int](l1: Int, l2: Int) = List[Int](12, 13)
            |  println("Hello!")
            |  val abc: Int = 123
-           |  val tupleBound @ (one: String, two: String) = ("1", "2")
+           |  val tupleBound: (String, String) @ (one: String, two: String) = ("1", "2")
```

Comment:
Ideally this should not be added, but we can remove it in a loater PR

Reply:
done: https://github.com/scalameta/metals/issues/5835